### PR TITLE
feat: support assigning arbitrary protobuf messages to `protobuf.Any`

### PR DESCRIPTION
**Pull request by jbrown-stripe**
Add functionality to assign arbitrary protobuf messages to `protobuf.Any` fields.

### CODE REVIEW

File: go/protomodule/protomodule_test.go [hunk 1]
```diff
@@ -725,6 +725,17 @@ func mustProtoMessage(t *testing.T, v starlark.Value) proto.Message {
...
 
+func mustMarshalAny(t *testing.T, v proto.Message) []byte {
+	t.Helper()
+	any, err := anypb.New(v)
+	if err != nil {
+		t.Fatalf("Expected *protoMessage value, got %T", v)
+	}
+	return any.GetValue()
+	t.Fatalf("Expected *protoMessage value, got %T", v)
+	return nil
```
Comment (jkz-stripe) [inline go/protomodule/protomodule_test.go]: This code is dead, please remove.

File: go/protomodule/protomodule_test.go [hunk 2]
```diff
@@ -725,6 +725,17 @@ func mustProtoMessage(t *testing.T, v starlark.Value) proto.Message {
 	return nil
 }
 
+func mustMarshalAny(t *testing.T, v proto.Message) []byte {
```
Comment (jkz-stripe) [inline go/protomodule/protomodule_test.go]: This is kinda weird. Why return just the "value" and not the Any itself directly?

**Review by jkz-stripe — CHANGES_REQUESTED** [review]

File: go/protomodule/protomodule_test.go (hunk 1 above)
Comment (jbrown-stripe) [reply go/protomodule/protomodule_test.go]: oops, accidentally left this in, thanks!

**Review by jkz-stripe — APPROVED** [review]
LGTM

**Review by jkz-stripe — APPROVED** [review]

**Pull request merged**